test/test-framework/build/bin/entrypoint

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
```

Comment:
Should this not be on a new line and `set -e` like the `user_setup`?

Reply:
Both are correct, this way is just a bit more compact I guess. I am just using what we generate in our scaffolds: https://github.com/operator-framework/operator-sdk/blob/master/internal/pkg/scaffold/entrypoint.go#L39:L51